spec/patches.md

```diff
@@ -24,25 +24,33 @@ queue=].
 With: For each [=event loop=], every [=task source=] that is not a [=scheduler task source=] must be
 associated with a specific [=task queue=].
 
+Add: An [=event loop=] has a numeric <dfn for="event loop">next enqueue order</dfn> which is
+initialized to 1.
```

Comment:
Ah, yeah it's optional if you choose to use timestamps. I chose to use an enqueue order since it's easy to understand and matches Chrome's implementation. I also removed the note about using a timestamp since I'm not sure how much that adds -- it's clear in the spec where the value is set, and implementations in general can make different choices as long as the observable behavior is the same.